sdk/src/wallet/bindings/nodejs/examples/30a-mint-native-token.js

```diff
@@ -0,0 +1,34 @@
+/**
+ * This example will mint native tokens
+ */
+const getUnlockedManager = require('./account-manager');
+
+async function run() {
+    try {
+        const manager = await getUnlockedManager();
+
+        const account = await manager.getAccount('0');
+
+        await account.sync();
+
+        // Get a tokenId from your account balance after running example
+        // 22-create-native-token.js
+        let tokenId =
+            '0x08e6210d29881310db2afde095e594f6f006fcdbd06e7a83b74bd2bdf3b5190d0e0200000000';
```

Comment:
Get the first foundry output as in the Rust example?

Reply:
Actually should be done in another PR, opened an issue https://github.com/iotaledger/iota-sdk/issues/691

Reply:
I feel like this is out of scope for this PR tbh.